string name
bool wait
---
bool success
string message
